### PR TITLE
Update artdaq-core-mu2e library names to match updated names in

### PR DESCRIPTION
**Pull request by eflumerf**
artdaq_core_mu2e v1_09_02

### CODE REVIEW

**Review by kutschke — APPROVED** [review]

**Comment by rlcee** [comment]
We will need to immediately turn around and update the envset for pedantic BTrk version.  @kutschke do you want to keep that a separate step or try to merge that in here?

**Comment by kutschke** [comment]
> We will need to immediately turn around and update the envset for pedantic BTrk version. @kutschke do you want to keep that a separate step or try to merge that in here?

Good catch.  I will ask Mackenzie to include it in her G4 update envset which should come later today.

File: Analyses/src/SConscript
Comment (pavel1murat) [inline Analyses/src/SConscript:1]: I wonder it that was sufficient for linking - I had to link an analysis module earlier today and had to add one more library:
 
                  'artdaq-mu2e-core_Overlays',
                  'artdaq-mu2e-core_Data',
                  'artdaq-core_Data',
                  'DTCInterface'

However , if the linker finds everything it needs w/o libartdaq-mu2e-core_Data.so, so be it

Comment (kutschke) [reply Analyses/src/SConscript:1]: Did you have a link-time or run-time error?

**Review by pavel1murat — APPROVED** [review]
looks good, I made a minor comment, but it may not require any action...

File: Analyses/src/SConscript
Comment (pavel1murat) [reply Analyses/src/SConscript:1]: yes

Comment (kutschke) [reply Analyses/src/SConscript:1]: Thanks.  Since the CI links I will go ahead and merge.

**Pull request merged**